### PR TITLE
Use the global text filter selector from CRUD

### PR DESCRIPTION
**Pull request by bipuladh**

### CODE REVIEW

File: frontend/packages/ceph-storage-plugin/integration-tests/tests/ocp-tests/installFlow.scenario.ts [hunk 1]
```diff
@@ -31,7 +31,7 @@ const testNodeLabel = (node) => {
   expect(hasOCSTaint(nodeJSON) || !hasTaints(nodeJSON)).toBe(true);
 };
 
-describe('Testing OCS Cluster Creation', () => {
+xdescribe('Testing OCS Cluster Creation', () => {
```
Comment (spadgett) [inline frontend/packages/ceph-storage-plugin/integration-tests/tests/ocp-tests/installFlow.scenario.ts:34]: We should open a JIRA issue or Bugzilla bug to track reenabling. (Generally we put this in this title of the disabled test to make it easy to find.)

**Comment by bipuladh** [comment]
https://bugzilla.redhat.com/show_bug.cgi?id=1822744

**Pull request merged**